src/components/capture/useDiscoverCapture.ts

```diff
@@ -264,16 +170,22 @@ function useDiscoverCapture(
                 },
                 (payload: any) => {
                     if (payload.error === JOB_STATUS_POLLER_ERROR) {
-                        jobFailed(payload.error);
+                        jobFailed(DEFAULT_POLLER_ERROR);
                     } else {
-                        jobFailed(`${messagePrefix}.test.failedErrorTitle`);
+                        jobFailed({
+                            title: 'discovery.failed.title',
+                            error: {
+                                message: 'discovery.failed.message',
+                            },
+                        });
```

Comment:
`Cleanup jobFailed`
Now have a default job status poller error settings object. Eventually I want to get almost all the `jobFailed` functions shared but that was a bit too much with this PR.